app/src/main/java/com/framgia/forder/data/source/remote/api/service/FOrderApi.java

```diff
@@ -131,4 +132,31 @@
     @POST("v1/orders_product_all")
     Observable<OrderCartResponse> requestOrderProduct(@Query("domain_id") int domainId,
             @Body OrderRequest orderRequest);
+
+    @PUT("v1/dashboard/order_products/0")
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "v1/dashboard/order_products/0" appears 7 times in this file; the first occurrence is on line 136
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals